projects/vrpano.md

```diff
@@ -0,0 +1,95 @@
+<!--brainstorming, fundraising, under development, beta, shipped, etc-->
+- [doing] Beta
+- [done] Demo
+
+## Previews
+<!--Add some screenshots to give a preview of your product-->
+![Vrpano Pre1](http://106.12.254.207:8006/ipns/bafzbeig6wke4thvbv3bbydhb6eflyt5w3iwxo4vb3lr5h7i5jwkzwcm22m/screenshot/vrshow1.png)
+![Vrpano Pre2](http://106.12.254.207:8006/ipns/bafzbeig6wke4thvbv3bbydhb6eflyt5w3iwxo4vb3lr5h7i5jwkzwcm22m/screenshot/vrshow2.png)
+![Vrpano Pre3](http://106.12.254.207:8006/ipns/bafzbeig6wke4thvbv3bbydhb6eflyt5w3iwxo4vb3lr5h7i5jwkzwcm22m/screenshot/vrshow3.png)
+
+## Use of IPFS, Filecoin and Libp2p
+<!-- Describe how your project uses any or all of these technologies, and why. -->
+**How do we plan to use IPFS and Filecoin?**
+
+- We provide more than just retrieval, we also provide IPFS to HTTP traffic interface.
+- Users can directly access VR and AR scene display through IPFS, but also through the traditional HTTP retrieval access.
+
+**What's our services?**
+
+We provide 3D data storage management, transfering, and displaying service. We support varies of formats for 3d data, such as:
+- Panoramic Data
+- VR Movie Data
+- AR Model Data
+- 3D Character Model Data
+
+## Target Audience
+<!--Describe who will be your project's users-->
+Photography studio, Panorama SaaS, Online Gallery, VR Studio etc
+
+## Rough estimated user base (if applicable)
+<!--How many users do you have right now?-->
+Our users are mainly in China, and 44GB of data has been Pined on FFS/IPFS.
+It is well known that Jingdezhen is the capital of porcelain, so our main customer is Jingdezhen Ceramic Museum, Jingdezhen Ceramic Industrial Park, Tao Xichuan Creative Park.
+The solution has been proven to be popular with users, and we will continue to iterate and upgrade to provide better service.
+
+
+## Github repo
+<!--Attach a link to your GitHub repo - open source is required - please make sure your repo has a license file and is licensed using MIT open source license! -->
+[https://github.com/ipfslab/vrpano)](https://github.com/ipfslab/vrpano)
+
+
+## Website
+<!--Link your website if available-->
+- Beta Links: http://vr.fmode.cn
+
+## Docs
+<!--Including a link to your project docs!-->
+- We are preparing product document.
+[Our documentation](https://github.com/ipfslab/vrpano/blob/master/README.md)
+
+## Team Info
+<!-- Introduce your amazing team - how many team members are working on this project and who are they?-->
+
+### Team Size  
+We are a team have more than 30 developers.
+We assigned 2 developers for this project.
+
+### Team members  
+
+[Ryane Liu](https://github.com/ryanemax/)  email: ryanemax@gmail.com 
+- CTO at Fmode Inc.
+- Project Lead
+- Project Planning and Customer Promotion
+
+[Zeyu Chen](https://github.com/xinyifly/) email: xinyifly@gmail.com
+- Professional Blockchain Engineer at Bitcoin China
+- Fullstack Engineer
+- FFS layer interface & 3d data management
+
+
+
+## How the community can engage
+* Email:  ipfsapp@yandex.com
```

Comment:
Can you create a discussion thread at https://github.com/ipfs/community/discussions and put the link here?